admin-dev/themes/default/template/controllers/employees/helpers/form/form.tpl

```diff
@@ -26,7 +26,7 @@
 
 {block name="field"}
 	{if $input.type == 'prestashop_addons'}
-		<div class="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
+		<div id="addons-connexion-container" class="col-lg-{if isset($input.col)}{$input.col|intval}{else}9{/if} {if !isset($input.label)}col-lg-offset-3{/if}">
```

Comment:
`id="addons-connexion-container"`

If possible, please change "connexion" to "connection"